FoodDeliveryUI/MainVIewModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;

```

Comment:
i believe this file is also unused, and not needed

Reply:
Done